### ISSUE TITLE
report output should be configurable on script and as a global configuration

### ISSUE THREAD

**Issue by danielo515** [shoreditch-ops/artillery#135]
It is a bit tedious to write a shell script just to test the different test scripts. It would be useful if you can specify the report output directly on the script or as a global config of artillery.

Regards

**Comment by hassy** [comment]
There's a `-o` option for both `artillery run` and `artillery quick` if that's what you mean?

**Comment by danielo515** [comment]
Yes, but you have to specify that option every time you call artillery. It is a bit uncomfortable, I would prefer to specify the output directory on the script configuration.

**Comment by hassy** [comment]
You can do something like `` artillery run -o "tmp/`date`.json" myscript.yaml ``

**Comment by danielo515** [comment]
Of course, but then I have to do it every time. I want to specify it once on the test script and forget about it

**Comment by danielo515** [comment]
Any intention to implement this?
